test/testdata/lsp/code_actions/extract_variable/method.rb

```diff
@@ -0,0 +1,25 @@
+# typed: true
+# selective-apply-code-action: refactor.extract
+
+def b; 1 + 123; end
+#          ^^^ apply-code-action: [B] Extract Variable
```

Comment:
I don't think this matters because these letters are just opaque tokens, but it looks like you're missing `[A]` in this file. not sure how much that matters to you.